src/Spectre.libGenetic.Tests/CrossoverOperatorTest.cpp

```diff
@@ -153,4 +161,15 @@ TEST_F(CrossoverOperatorTest, cuts_are_from_uniform_distribution)
         EXPECT_LT(count, meanCount + allowedCountMiss) << i;
     }
 }
+
+TEST_F(CrossoverOperatorTest, test_if_return_first_individual_if_fillup_is_too_low)
```

Comment:
Missing opposite case with assumptions against situation, when fillup is too high.

Reply:
done